snapcraft/internal/elf.py

```diff
@@ -175,6 +176,14 @@ def _extract(self, path):  # noqa: C901
         with open(path, 'rb') as fp:
             elf = elftools.elf.elffile.ELFFile(fp)
 
+            # A set of fields to identify the architecture of the ELF file:
```

Comment:
Thanks for adding a comment, can you order these as in the tuple and also add the example value of the constant for x86-64 in e_machine and so on and so forth.